src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ISymbolExtensions.RequiresUnsafeModifierVisitor.cs

```diff
@@ -71,6 +71,20 @@ public override bool VisitPointerType(IPointerTypeSymbol symbol)
                 return true;
             }
 
+#if !CODE_STYLE
+
+            public override bool VisitFunctionPointerType(IFunctionPointerTypeSymbol symbol)
+            {
+                if (!_visited.Add(symbol))
+                {
+                    return false;
+                }
```

Comment:
Following the pattern from VisitPointerType but it's unclear to me why we track visited here since this doesn't have to recurse at all.